core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -548,6 +543,8 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |  --conf, -c PROP=VALUE       Arbitrary Spark configuration property.
         |  --properties-file FILE      Path to a file from which to load extra properties. If not
         |                              specified, this will look for conf/spark-defaults.conf.
+        |  --extra-properties-files    Whether to load properties from conf/spark-defaults.conf,
```

Comment:
Not strong opinion. But this flag `--extra-properties-files` looks not like a boolean one. Maybe `load-default-properties-file`?

Reply:
At least, it should be `--extra-properties-file` I think, as it only loads one file.

Reply:
Let me update to `--load-spark-defaults`